changelog.md

```diff
@@ -11,6 +11,9 @@
 
 [//]: # "Additions:"
 
+- `std/jscore`:
+  + Added `toDateString`, `toGMTString`, `toISOString`, `toJSON`, `toTimeString`, `toUTCString` converters for `DateTime`.
```

Comment:
Wrong place to add changelog. It's https://github.com/nim-lang/Nim/blob/devel/changelogs/changelog_2_0_0.md instead

Reply:
Thanks, done.